docs/docs/get-started/setup-lightdash/get-project-lightdash-ready.mdx

```diff
@@ -34,20 +39,51 @@ Not sure what a .yml file is? Make sure to [check out dbt's docs](https://docs.g
 
 Here's an example of our `Community stats daily` model we've defined in dbt. In Lightdash, we can see it in our list of available Tables to explore.
 
-<img src={TablesView} width="920" height="391" style={{display: "block", margin: "0 auto 20px auto"}}/>
+<img
+    src={TablesView}
+    width="920"
+    height="391"
+    style={{ display: 'block', margin: '0 auto 20px auto' }}
+/>
 
 Now that we know what tables are, let's add some and create our first project!
 
 ---
 
 ## Step 1: Install the Lightdash CLI tool
 
-:::info
+<details>
```

Comment:
Is there a way to check that you have NodeJS/NPM installed?
I wonder if we can we make the first step of the instructions to `check that they have it installed`?

Reply:
> Is there a way to check that you have NodeJS/NPM installed? I wonder if we can we make the first step of the instructions to `check that they have it installed`?

@TuringLovesDeathMetal something like this?
<img width="916" alt="CleanShot 2022-10-24 at 17 26 52@2x" src="https://user-images.githubusercontent.com/962095/197537045-6efec458-9a38-43a0-9ec0-045ecdc88cbf.png">

Reply:
beautiful! 😚 👌